openslides/motions/views.py

```diff
@@ -395,8 +395,8 @@ def destroy(self, *args, **kwargs):
         """
         Customized view endpoint to delete a motion poll.
         """
-        result = super().destroy(*args, **kwargs)
         poll = self.get_object()
+        result = super().destroy(*args, **kwargs)
```

Comment:
If you call destroy first self.get_object() will fail since the object was destroyed.